src/components/Dashboard/Dashboard.jsx

```diff
@@ -48,6 +49,7 @@ export const Dashboard = props => {
         toggleSubmitForm={toggle}
         role={props.auth.user.role}
         leaderData={leaderData}
+        setCount={setCount}
```

Comment:
I think here you want to use 
`  setCount={weeklySummariesCount}`

correct me if i am wrong